bin/start

```diff
@@ -6,14 +6,16 @@ Test the secrets-provider-for-k8s image. This script sets up a Conjur cluster in
 container and a secrets-provider-for-k8s init container. Finally it tests that the outcome is as expected (for example,
 in the vanilla flow it verifies that the Conjur secret was provided to the app container via the environment.
 
-Usage: ./bin/test_integation [options]
+Usage: ./bin/start [options]
     --docker      Run the integation tests in a docker container. This is helpful when running the test in a machine that
                   doesn't have "oc" or "kubectl" installed on it.
 
     --demo        Run a full end-to-end demo with the pet-store demo. This demo deploys an environment with an app container
-                  that needs credentials to a DB from Conjur. After running "./bin/test_integation --demo" run "oc get routes"
+                  that needs credentials to a DB from Conjur. After running "./bin/start --demo" run "oc get routes"
                   and go to "<the exposed route>/pets" in a browser. The entry "Mr. Sidecar" should be displayed on the screen.
 
+    --dev         Run the development environment. By default the environment is run against a Conjur OSS deployment on GKE.
+
     --oss         Run the integration tests with a OSS Conjur deployment. By default the tests are run against a Conjur OSS deployment.
```

Comment:
will this also run with GKE by default? If so, please add it to the text (same for `--dap`)

Reply:
@sigalsax ^